lib/ansible/utils/display.py

```diff
@@ -58,19 +58,28 @@ def filter(self, record):
 
 
 logger = None
-# TODO: make this a logging callback instead
+# TODO: make this a callback event instead
 if getattr(C, 'DEFAULT_LOG_PATH'):
     path = C.DEFAULT_LOG_PATH
     if path and (os.path.exists(path) and os.access(path, os.W_OK)) or os.access(os.path.dirname(path), os.W_OK):
-        logging.basicConfig(filename=path, level=logging.DEBUG, format='%(asctime)s %(name)s %(message)s')
-        mypid = str(os.getpid())
-        user = getpass.getuser()
-        logger = logging.getLogger("p=%s u=%s | " % (mypid, user))
+        logging.basicConfig(filename=path, level=logging.INFO, format='%(asctime)s p=%(user)s u=%(process)d | %(message)s')
+        logger = logging.LoggerAdapter(logging.getLogger(__name__), {'user': getpass.getuser()})
```

Comment:
Should we just hardcode this instead of using `__name__`?  Using `__name__` could make this dynamic, as it could be `__main__` or `ansible.utils.display`.  We might benefit from just making it `ansible`

Reply:
im still debating this one, i initially had 'ansible' hardcoded, but though it should really match the specific cli used

Reply:
I think we should hard code this to be `ansible`. I'm not sure logging the module name will be useful to the end user if this is meant to behave more like a CLI logger and not an application debug logger.